### ISSUE TITLE
Rich workspace in grid view

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#6058]
- have a folder info (rich workspace)
- enable grid view
- should show folder info in first row

![2020-05-12-170644](https://user-images.githubusercontent.com/5836855/81710390-fbce3800-9472-11ea-89d6-d298826091d0.png)

**Comment by koushikchoudhury0** [comment]
> * have a folder info (rich workspace)
> * enable grid view
> * should show folder info in first row
> 
> ![2020-05-12-170644](https://user-images.githubusercontent.com/5836855/81710390-fbce3800-9472-11ea-89d6-d298826091d0.png)

Is the attacthed screenshot your expected UI? If so, may I'd like to contribute

**Comment by tobiasKaminsky** [comment]
No, the expected UI is to have the "Markwon" as first row spanned above all columns.
So that in second row you would see "share", "docu…5", mp3.

**Comment by tobiasKaminsky** [comment]
>  I'd like to contribute

That would be aweseome <3
Do not hesitate to ask, if you have any question.
